qtfred/src/ui/FredView.cpp

```diff
@@ -582,6 +620,10 @@ void FredView::on_actionBriefing_triggered(bool) {
 	auto eventEditor = new dialogs::BriefingEditorDialog(this);
 	eventEditor->show();
 }
+void FredView::on_actionWaypoint_Paths_triggered(bool) {
+	auto editorDialog = new dialogs::WaypointEditorDialog(this, _viewport);
```

Comment:
Linux and Windows CI builds are failing here. This seems like a perfectly valid line so I'm not sure what the issue is.